activerecord/test/cases/adapters/postgresql/enum_test.rb

```diff
@@ -158,26 +158,15 @@ def test_works_with_activerecord_enum
   end
 
   def test_enum_type_scoped_to_schemas
-    old_search_path = @connection.schema_search_path
-    @connection.create_schema("test_schema")
-    @connection.schema_search_path = "test_schema"
-    @connection.schema_cache.clear!
-    @connection.create_enum("mood_in_other_schema", ["sad", "ok", "happy"])
+    with_test_schema("test_schema") do
+      @connection.create_enum("mood_in_other_schema", ["sad", "ok", "happy"])
 
-    assert_nothing_raised do
-      @connection.create_table("postgresql_enums_in_other_schema") do |t|
-        t.column :current_mood, :mood_in_other_schema, default: "happy", null: false
+      assert_nothing_raised do
```

Comment:
I understand that this is the style used before, but `assert_nothing_raised` is too abstract, imo. Maybe better to test that the table exists?